build/azDevOps/azure/air-infrastructure-data-vars.yml

```diff
@@ -32,35 +21,11 @@ variables:
   # **`terraform_state_workspace: `**
   # avoid running anything past dev that is not on master
   # sample value: company-webapp
-  - name: tf_state_rg
-    value: Stacks-Ancillary-Resources
-  - name: tf_state_storage
-    value: amidostackstfstate
-  - name: tf_state_container
-    value: tfstate
   - name: tf_state_key
-    value: $(domain)
+    value: core-data
   - name: TF_FILE_LOCATION
     value: $(self_repo_tf_src)
 
-  # App Specific Taxonomy
-  - name: component
-    value: de
-  - name: role
```

Comment:
this actually isn't being referenced anywhere so this has been removed

Reply:
I raised this the other day to Andy, so there is a PR to bring this back in: https://github.com/Ensono/stacks-azure-data/pull/171/files

Docs also updated to reflect this being a value that is now used (added in anticipation)

Reply:
Hey,I think Component is getting used in resource name

Reply:
Sorry wasn't clear 'role' isn't being used so is removed. 'component' has been moved to the common-variables.yml and stacks-cli will inject the component value to it

Reply:
@Danphillipz I will close that draft PR as this refactor includes the fix.